docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,675 @@
+---
+title: "Movement Sensor Component"
+linkTitle: "Movement Sensor"
+weight: 70
+type: "docs"
+description: "A sensor that measures location, kinematic data, or both."
+tags: ["movement sensor", "gps", "imu", "sensor", "components"]
+icon: "img/components/imu.png"
+no_list: true
+aliases:
+    - /components/movement-sensor/
+# SME: Rand
+---
+
+The movement sensor component is an abstraction of a sensor that gives data on where a robot is and how fast it is moving.
+
+We have chosen to abstract these types of sensors into one common API.

```

Comment:
This sounds a bit like we're narrating product choices in the docs - that's not adding anything useful
```suggestion
```

Reply:
True! I should've edited this section more, oops.